.github/workflows/tests.yml

```diff
@@ -18,6 +18,7 @@ jobs:
         run: |
           python -m pip install --upgrade pip
           pip install .[test]
+          pip install -e .

```

Comment:
do we need both this and above line? sounds a bit redundant (but I'm not sure)

Reply:
Not sure, i can try remove the above one